app/presenters/courses_presenter.rb

```diff
@@ -59,6 +59,10 @@ def can_remove_course?
     @can_remove ||= current_user&.admin? || campaign_organizer?
   end
 
+  def can_delete_course?
+    @can_remove ||= current_user&.admin? || campaign_organizer?
```

Comment:
Is this variable name a mistake? It looks like this is just a complete duplicate of `can_remove_course?`

Reply:
I think a little more details will be required to determine whether a course can be deleted. For any given course, it may be in multiple campaigns, but normally only one of the facilitators/instructors for the program itself is allowed to delete a course.

I think it also makes sense to allow an campaign organizer or admin to delete a course that is in campaign... but only if it is not in any other campaigns. Otherwise, it should probably just remove the campaign and report that the course could not be deleted because it is part of another campaign.

Reply:
I have updated `@can_remove` to `@can_delete`.

Still it's unclear to me what permission changes are required. I'll give you a little rundown, and then can you please mention the changes once again?

### Workflow 
The Remove and Delete button is only visible to a user if 
```
  def can_delete_course?
    @can_delete ||= current_user&.admin? || campaign_organizer?
  end
```
is true.


When clicked on the Remove and Delete button. 
![image](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/92005669/9404b59a-7f9b-42e1-910b-5767f9f2cbbc)

It checks if the program is only present in one campaign.

If yes. it removes and deletes the program.

If not? then it shows - 
![image](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/92005669/630c4140-f5c2-43d9-805d-bac7ecf63ed9)

### I set the same level of permission to - _ability to delete a program_ as the permissions given to -  _ability to remove a program from campaign_.

### But the **program can only be deleted** if it is not present in any other campaign.

Reply:
This flow makes sense to me.

Reply:
But, if a user tries to 'Remove and Delete' and it is in another campaign, it should still remove this campaign and the notification should reflect that.